### ISSUE TITLE
[Bug] pr not being created after step 3

### ISSUE THREAD

**Issue by thdotnet** [skills/hello-github-actions#31]
I am following the steps of this skill, but after the step # 3, I was expecting a new PR to be created, but it didn't happen

**Comment by sinsukehlab** [comment]
The similar issue: #30

**Comment by cmwilson21** [comment]
Thanks for the report. Closing this in favor of https://github.com/skills/hello-github-actions/issues/30. As stated above, they are duplicate issues. 💛
